src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -5831,11 +5840,45 @@ namespace InternalHeatGains {
                 }
             }
             TDPAirIn = PsyTdpFnWPb(WAirIn, StdBaroPress, RoutineName);
-            RHAirIn = PsyRhFnTdbWPb(TAirIn, WAirIn, StdBaroPress, RoutineName);
+            RHAirIn = 100.0 * PsyRhFnTdbWPb(TAirIn, WAirIn, StdBaroPress, RoutineName); // RHAirIn is %
```

Comment:
@xuanluo113 Moved this correction here so that the relative humidity range outputs are also correct. This should result in more diffs in the test suite data center files.